src/main/java/net/wurstclient/hacks/FlightHack.java

```diff
@@ -48,24 +68,38 @@ public void onDisable()
 	{
 		WURST.getEventManager().remove(UpdateListener.class, this);
 		WURST.getEventManager().remove(IsPlayerInWaterListener.class, this);
+
 	}
-	
+	int ticks = 0;

```

Comment:
This field should probably be reset to zero in the onEnable() method. Setting the value here will only set it once (when the game is started) and then it will just keep going forever.